examples/training/datasets/README.md

```diff
@@ -0,0 +1,56 @@
+# Training Datasets
+
+Most dataset configurations will take one of four forms (below you will see examples of each case):
+
+- Case 1: The example is a pair of sentences and a label indicating how similar they are. The label can be either an integer or a float. This case applies to datasets originally prepared for Natural Language Inference (NLI), since they contain pairs of sentences with a label indicating whether they infer each other or not.
```

Comment:
```suggestion
- **Case 1**: The example is a pair of sentences and a label indicating how similar they are. The label can be either an integer or a float. This case applies to datasets originally prepared for Natural Language Inference (NLI), since they contain pairs of sentences with a label indicating whether they infer each other or not.
   **Case Example:** [SNLI](https://huggingface.co/datasets/snli).
```